mentors/prolog/t.json

```
@@ -6,5 +6,13 @@
     "link_url": "https://tiago.org",
     "avatar_url": "http://0.gravatar.com/avatar/9d97b401cb8272790d9ef0ebf27d0935",
     "bio": "Did Prolog professionally a long time ago. Still think its the best language in the world!"
-  }
+  },
+  {
+  "github_username": "teto1992",
+  "name": "Stefano Forti",
+  "link_text": "Personal site",
+  "link_url": "http://pages.di.unipi.it/forti/",
+  "avatar_url": "http://pages.di.unipi.it/forti/assets/images/profile.jpg",
+  "bio": "Post-doc researcher and Pisa CoderDojo founder and mentor. I use Prolog for fast (and clean!) prototyping of my research and fun ideas. I really enjoy mentoring people in improving their programming skills!"
+}
```

```suggestion
    "github_username": "teto1992",
    "name": "Stefano Forti",
    "link_text": "Personal site",
    "link_url": "http://pages.di.unipi.it/forti/",
    "avatar_url": "http://pages.di.unipi.it/forti/assets/images/profile.jpg",
    "bio": "Post-doc researcher and Pisa CoderDojo founder and mentor. I use Prolog for fast (and clean!) prototyping of my research and fun ideas. I really enjoy mentoring people in improving their programming skills!"
  }
```
Just a minor change request.